src/main/resources/templates/db_scripts/schema.json

```diff
@@ -79,10 +79,32 @@
       ]
     },
     {
-      "tableName": "acquisitions_units",
+      "tableName": "acquisitions_unit",
       "fromModuleVersion": 6.1,
       "withMetadata": true,
       "populateJsonWithId": true
+    },
+    {
+    "tableName": "acquisitions_unit_membership",
+      "fromModuleVersion": 6.1,
+      "withMetadata": true,
+      "populateJsonWithId": true,
+      "foreignKeys": [
+        {
+          "fieldName": "acquisitionsUnitId",
+          "targetTable": "acquisitions_unit"
+        }
+      ],
+      "index": [
+         {
+            "fieldName" : "userId",
+            "tOps": "ADD"
```

Comment:
According to RMB documentation tOps is optional for all RMB index types and defaults to ADD the index. So you can remove it if you will see it fit.

Reply:
Done